### DISCUSSION TITLE
CSS not updating in preview

### DISCUSSION THREAD

**Discussion by mintonan** [Budibase/budibase#12189] [Help] [answered]
I am making some edits to the homescreen of my app for users to navigate but am running into some small issues

When trying to update the CSS in either the right side panel or inline styling, it shows changes being made in the development window but when I hit the preview button it goes back to as if I haven't made any changes at all. 

Before Changes:
![image](https://github.com/Budibase/budibase/assets/124909937/c4ce41fb-c9d3-4970-95e9-6187dda385f9)

After Changes:
![image](https://github.com/Budibase/budibase/assets/124909937/2282b121-d5ab-4150-b2cd-3663867f1ecf)

Preview Screen After Changes:
![image](https://github.com/Budibase/budibase/assets/124909937/9a882a5f-d28b-4ed1-8647-1bfddcc4ca9c)

Here is my Components List as well:
![image](https://github.com/Budibase/budibase/assets/124909937/9e1c3a56-be57-4460-a83f-d0ede49ecbf9)

I have tried changes within a container and without but it still keeps reverting. Is there something about the Cards or repeater component component that I am missing?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @mintonan,

Can you try using `px` at the end of your width; `400px` or using `%` obviously not 400% in this case. Some browser don't know what to do whenever a non specific width is added. So they ignore it instead. Additionally, this could be related to how you are flexing the columns. 

Could you take a screenshot were you have selected your repeater to see what settings you are using there. (I've included a screenshot below.)
<img width="684" alt="Screenshot 2023-10-30 at 10 12 44" src="https://github.com/Budibase/budibase/assets/126772285/ed1a24e1-3a7c-47ec-9f8c-134e28988c57">

If the above doesn't resolve it, could you export a simple app with the issue replicated and I'll take a more in-depth look.

Thanks!

**Reply by mintonan** [reply]
Thanks @ConorWebb96 ! Adding the px worked. I swear I thought I tried that but it must have just been one of those things I skipped. Will definitely keep it more in mind from now on
